mobile/src/actions/mediaBackup.js

```diff
@@ -33,8 +45,11 @@ export const mediaBackup = (dir) => async (dispatch, getState) => {
     let photos = await getFilteredPhotos()
     const alreadyUploaded = getState().mobile.mediaBackup.uploaded
     const dirID = await getDirID(dir)
+    const totalUpload = photos.length - alreadyUploaded.length
```

Comment:
I needed to get the total of *going to be uploaded* photos.

I think that 
```js
photos.length - alreadyUploaded.length === photos.filter(photo => !alreadyUploaded.includes(photo.id)).length
```

So this remark, though good enough, is just a matter of refactoring.